TodoList/src/app/screens/Footer/index.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { TouchableOpacity, Text, StyleSheet } from 'react-native'
+import { styles } from './styles.js';
+
+export default class Footer extends Component {
```

Comment:
`Footer` should be a function. It doesn't have internal state and doesn't rely on lifecycle functions